source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -965,13 +988,21 @@ def create_next_kraken_request(self, request, responses):
 
         to do that we find ask the next (resp previous) query datetime
         """
-        vjs = journey_filter.get_qualified_journeys(responses)
-        if request["clockwise"]:
-            request['datetime'] = self.next_journey_datetime(vjs, request["clockwise"])
+
+        # If Kraken send a new request date time, we use it
+        # for the next call to skip current Journeys
+        if len(responses) > 0 and responses[0].HasField('next_request_date_time'):
```

Comment:
minor: if responses  and responses[0] ...

Reply:
:white_check_mark: